gensrc/thrift/PlanNodes.thrift

```diff
@@ -967,6 +1021,12 @@ struct TPlanNode {
   63: optional TLakeScanNode lake_scan_node;
   
   64: optional TNestLoopJoinNode nestloop_join_node;
+
+  // 70 ~ 80 are reserved for stream operators
+  // Stream plan
+  70: optional TBinlogSourceNode binlog_source_node;
```

Comment:
`StreamingScanNode` is better?  `binlog` may not the only source for streaming?

Reply:
ok.